site/pages/api/create-consequence-operator.api.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Rather than disable these rules we could create a new Request interface to pass in:
```ts
interface OperatorConsequenceRequest extends NextApiRequest {
    body: OperatorConsequence & {
        consequenceOperators: string;
    };
}

const createConsequenceOperator = async (req: OperatorConsequenceRequest, res: NextApiResponse): Promise<void> => {
```

Reply:
Nice, added.